rdr_service/api/biobank_specimen_api.py

```diff
@@ -0,0 +1,36 @@
+from flask import request
+
+from rdr_service.api.base_api import UpdatableApi
+from rdr_service.api_util import HEALTHPRO
+from rdr_service.app_util import auth_required
+from rdr_service.dao.biobank_specimen_dao import BiobankSpecimenDao
+from rdr_service.model.utils import from_client_participant_id
+from werkzeug.exceptions import BadRequest
+
+
+class BiobankSpecimenApi(UpdatableApi):
+    def __init__(self):
+        super().__init__(BiobankSpecimenDao(), get_returns_children=True)
+
+    @auth_required(HEALTHPRO)
+    def post(self, p_id):
+        return super().post(participant_id=p_id)
+
+    @auth_required(HEALTHPRO)
+    def get(self, p_id=None, bo_id=None):
+        return super(BiobankSpecimenApi, self).get(id_=bo_id, participant_id=p_id)
+
+    @auth_required(HEALTHPRO)
+    def put(self, *args, **kwargs):  # pylint: disable=unused-argument
+        resource = request.get_json(force=True)
+
+        for required_field in ['rlimsID', 'orderID', 'testcode', 'participantID']:
+            if required_field not in resource:
+                raise BadRequest("Missing field: %s" % required_field)
+
+        participant_id = from_client_participant_id(resource['participantID'])
```

Comment:
I just double confirmed one thing, for the participant ID in the payload, it's biobank's participant id, in our system, we call it biobankID, it's one on one mapping with our participantID.
Here is the RLIMS field mapping: https://docs.google.com/spreadsheets/d/1oBwWN5It5SESITmuIVrL-YjzorVDRyD8/edit#gid=804532217

And for this biobankID, in RDR it's 9 digits, but outside RDR, we add a prefix for it. 
Here is the detail: https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/model/config_utils.py#L9

Sorry that I jus realize this when I read your code.